go.mod

```diff
@@ -1,8 +1,16 @@
-module github.com/abrander/garmin-connect
+module github.com/garmin-connect
 
-go 1.15
+go 1.20
```

Comment:
Is Go 1.20 needed for this fix?

Reply:
No i was just moving it to 1.20 as there are some tls security fixes so feel free to remove it